etl/xml_importer/entities/artwork.py

```diff
@@ -72,8 +88,11 @@ def _parse_inception(self):
     def _parse_inscription(self):
         self.inscriptions = []
         all_inscriptions = self.lido.findall(paths["Artwork_Inscription_Path"], namespace)
-        for inscription in all_inscriptions:
-            self.inscriptions.append(inscription.text)
+        if all_inscriptions is not None:
```

Comment:
This check is not necessary here since `findall(...)` returns an empty list if nothing was found, which can be safely iterated.
```suggestion
        for inscription in all_inscriptions:
            self.inscriptions.append(inscription.text)
```